redash/settings/__init__.py

```diff
@@ -123,7 +123,7 @@ def all_settings():
 
 HOST = os.environ.get('REDASH_HOST', '')
 
-ALERTS_DEFAULT_MAIL_SUBJECT_TEMPLATE = os.environ.get('REDASH_ALERTS_DEFAULT_MAIL_SUBJECT_TEMPLATE', "({state}) {alert_name}")
+ALERTS_DEFAULT_MAIL_SUBJECT_TEMPLATE = os.environ.get('REDASH_ALERTS_DEFAULT_MAIL_SUBJECT_TEMPLATE', u"({state}) {alert_name}")
```

Comment:
I think you'll also want to `decode('utf-8')` the value in case `REDASH_ALERTS_DEFAULT_MAIL_SUBJECT_TEMPLATE` is indeed set since it would be a bytestring on Python 2.

Speaking of which, is this the right time to ask about Python 3? 😬

Reply:
> Speaking of which, is this the right time to ask about Python 3? 😬

:-) there is actually a pull request to update the code to be compliant with Python 3. But I'm concerned on how to test everything, specially the different query runners?